sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -334,8 +334,8 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
     case ShowNamespaces(ResolvedNamespace(catalog, ns), pattern, output) =>
       ShowNamespacesExec(output, catalog.asNamespaceCatalog, ns, pattern) :: Nil
 
-    case ShowTables(ResolvedNamespace(catalog, ns), pattern, output) =>
-      ShowTablesExec(output, catalog.asTableCatalog, ns, pattern) :: Nil
+    case r @ ShowTables(ResolvedNamespace(catalog, ns), pattern) =>
+      ShowTablesExec(r.output, catalog.asTableCatalog, ns, pattern) :: Nil
 
```

Comment:
r -> s?

Reply:
If you look at the code around, you can notice `r` everywhere. `r` means resolved I guess.

Reply:
> If you look at the code around, you can notice `r` everywhere. `r` means resolved I guess.

All right, since in this class, `ShowXXX` always use `s`.

Reply:
Removed it since the alias is not needed anymore.